Source/JavaScriptCore/wasm/js/WebAssemblyFunction.h

```diff
@@ -105,6 +106,8 @@ class WebAssemblyFunction final : public WebAssemblyFunctionBase {
 #if ENABLE(JIT)
     CodePtr<JSEntryPtrTag> m_jsToWasmICJITCode;
 #endif
+
+    friend JSC::LLIntOffsetsExtractor;
```

Comment:
In most of cases, we define it at the head of the class / struct definition. Can you move it?